__tests__/index.test.js

```diff
@@ -197,8 +199,88 @@ describe("/api/articles/:article_id/comments tests", () => {
       .get("/api/articles/nine/comments")
       .expect(400)
       .then((response) => {
-        console.log(response.body.message);
         expect(response.body.message).toBe("Bad request.");
       });
   });
 });
+
+describe("POST:/ api/articles/:article_id/comments tests", () => {
+  test("201: posts a new comment for an article", () => {
+    const newComment = {
+      username: "rogersop",
+      body: "yes, it's a comment",
+    };
+    return request(app)
+      .post("/api/articles/9/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        expect(response.body.comment.author).toBe("rogersop");
+        expect(response.body.comment.body).toBe("yes, it's a comment");
+        expect(response.body.comment.article_id).toBe(9);
```

Comment:
There is a possibility that test data might get changed and this id won't be 9. We can assert that the property is here and it's type but not its actual value to stop this test being brittle.